CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.20.5 (unreleased)
+
+* *(types)* Added support for MSC2530 media event content
```

Comment:
The changelog shouldn't be updated in PRs (I should probably write contributing instructions at some point 🤔)

Reply:
Okay, I'll remove it. I thought I was being helpful 😆